Quran/HighlightingView.swift

```diff
@@ -8,12 +8,22 @@
 
 import UIKit
 
+@objc protocol HighlightingViewDelegate {
```

Comment:
That's true, but if you need to do it just for swift, you can use `class` constraint. [Explained here](http://stackoverflow.com/questions/24066304/how-can-i-make-a-weak-protocol-reference-in-pure-swift-w-o-objc).